components/Act/ActDetailForm.js

```diff
@@ -39,7 +39,7 @@ class ActDetailForm extends Component {
     }
 
     this.setImgUrl = this.setImgUrl.bind(this)
-    this.change = this.change.bind(this)
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
Hi Andrew, the handle change function use arrow function so I think we don't need to bind the function in the constructor anymore.